tls/s2n_key_update.c

```diff
@@ -20,9 +20,13 @@
 #include "tls/s2n_tls13_handshake.h"
 #include "tls/s2n_record.h"
 
+#include "crypto/s2n_sequence.h"
+
 #include "utils/s2n_safety.h"
 
 int s2n_key_update_write(struct s2n_blob *out);
+int s2n_check_key_limits(struct s2n_connection *conn, struct s2n_blob *sequence_number); 
```

Comment:
What does this do? That is not a clear name.

Reply:
This checks to see if we need to rekey. I guess it would be more accurate to name it "check_record_limit()" now that we are using a record limit threshold.